apiserver/src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java

```diff
@@ -168,7 +168,6 @@ public Project getProject(final String name, final String version) {
         final Map<String, Object> params = filterBuilder.getParams();
 
         preprocessACLs(query, queryFilter, params);
-        query.setFilter(queryFilter);
```

Comment:
This was a bug. Setting the filter here again caused the filter modified by `preprocessACLs` to be overwritten, which bypassed the ACL check.